packages/block-editor/src/components/link-control/style.scss

```diff
@@ -22,6 +23,22 @@ $preview-image-height: 140px;
 		width: 90vw;
 		max-width: $modal-min-width;
 	}
+
+	.show-icon-labels & {
+		#{$root}__tools {
```

Comment:
We don't need it to be this specific, it's fine to nest `.components-button.has-icon` directly under `.show-icon-labels &`. The decreased specificity also has the advantage that if we ever add another icon button to the link control, text label should just work for it out of the box 🙂